Cannot open "currentTask"

It's not possible to open the current task anymore via the bell icon.
![openCurrentTask](https://user-images.githubusercontent.com/22615787/81558260-a9671b80-938d-11ea-82a4-b404e2c9c35d.gif)

Stacktrace:
`Caused by: java.lang.NullPointerException
	at org.kitodo.data.database.beans.User.<init>(User.java:119)
	at org.kitodo.production.services.data.UserService.getCurrentUser(UserService.java:208)
	at org.kitodo.production.controller.SessionClientController.getAvailableClientsOfCurrentUser(SessionClientController.java:148)
	at org.kitodo.production.controller.SessionClientController.userHasOnlyOneClient(SessionClientController.java:61)
	at org.kitodo.production.controller.SessionClientController.getCurrentSessionClientName(SessionClientController.java:47)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)`

Somehow no authenticated User is been found

--- a/Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java
+++ b/Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java
@@ -120,6 +120,9 @@ public class Process extends BaseTemplateBean {
     private boolean exported;
 
     @Transient
+    private User blockedUser;
+
+    @Transient
     private List<Map<String, Object>> metadata;
 
     /**
@@ -500,6 +503,25 @@ public class Process extends BaseTemplateBean {
      */
     public void setMetadata(List<Map<String, Object>> metadata) {
         this.metadata = metadata;
+    }
+
+    /**
+     * Get blocked user.
+     *
+     * @return User object if this user is blocked
+     */
+    public User getBlockedUser() {
+        return blockedUser;
+    }
+
+    /**
+     * Set blocked user.
+     *
+     * @param blockedUser
+     *            User object
+     */
+    public void setBlockedUser(User blockedUser) {
+        this.blockedUser = blockedUser;
     }
 
     /**

